hs-bindgen/fixtures/distilled_lib_1.pp.hs

```diff
@@ -30,6 +30,8 @@ a_DEFINE_2 = 2
 tWO_ARGS :: forall a0. P.Integral a0 => a0
 tWO_ARGS = 13398
 
+foreign import capi safe "distilled_lib_1.h some_fun" some_fun :: (F.Ptr CATypeT) -> CUint32T -> Void -> IO CInt32T
```

Comment:
I noticed that parameter `uint8_t k[]` is being translated do `Void`.  Array parameters are rewritten as pointers, so perhaps we would like to translate to `F.Ptr CUint8T` if not `[CUint8T]`?  I think that the list translation would be preferred, however, so that a null pointer cannot be used.

Perhaps this is a detail that will be addresses in a separate/future PR?

Reply:
https://github.com/well-typed/hs-bindgen/blob/main/hs-bindgen/src/HsBindgen/C/Fold/Type.hs#L290 added in #330